.github/workflows/build.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           path: ${{ github.workspace }}/packages/*.tar.gz
 
   macos-cross-build:
-    runs-on: macos-latest
+    runs-on: macos-10.15
```

Comment:
@gnufede mentioned that macos support for arm64 started with big sur (11), so you can probably revert this one to `macos-latest`.

Reply:
You can also use `macos-11.0` if you want to have it pinned, IIRC

Reply:
I didn't check the details but it probably compiles because we cross-compile here.

Reply:
Pinned to macos-11 anyway 👍